src/flashbar/collapsible.scss

```diff
@@ -241,7 +241,7 @@ the grid layout will be:
     row-gap: 0;
 
     > .header {
-      font-weight: awsui.$font-button-weight;
+      font-weight: awsui.$font-weight-button;
```

Comment:
![Screenshot 2024-11-19 at 09 20 06](https://github.com/user-attachments/assets/8dc76d41-43b5-48d0-a7e2-4fbfc9fadca5)

unsure on the semantics here, I guess it _kinda_ is a button?

Reply:
Agreed I'm inclined to leave this as button.